crates/red_knot_python_semantic/src/semantic_index/use_def/symbol_state.rs

```diff
@@ -436,7 +438,7 @@ mod tests {
         }
 
         pub(crate) fn assert_declarations(&self, may_be_undeclared: bool, expected: &[u32]) {
```

Comment:
Are these methods used outside this file? If not, can we make them private?

I generally prefer standalone functions or newtype wrapper around adding test specific methods to a type. They polute the type's namespace (I e.g. don't know if rust analyzer filters the methods out when outside a test).